addons/website_slides/models/slide_channel.py

```diff
@@ -722,6 +726,23 @@ def _remove_membership(self, partner_ids):
         if removed_channel_partner_domain:
             self.env['slide.channel.partner'].sudo().search(removed_channel_partner_domain).unlink()
 
+    def _send_share_email(self, emails, fullscreen):
```

Comment:
"fullscreen" parameter is not necessary here (since you will split the controller methods).